documentation/meta/ci_cd/actions.md

```diff
@@ -36,10 +36,10 @@ By default, both Python and Node.js are installed and configured by this action
 but it's recommended to use `setup_python` and `setup_nodejs` inputs to only
 install the runtime needed by a job.
 
-By default, it also runs `just install` so that the core dependencies needed on
-the system are installed. This recipe can be customised via the `install_recipe`
-input (or even disabled with a blank string, to speed up the process in case no
-host dependencies are needed).
+By default, it also runs `./ov just install` so that the core dependencies
```

Comment:
This refers to the CI, which does not currently use `ov` (see #4406).

```suggestion
By default, it also runs `just install` so that the core dependencies
```